torch/csrc/jit/script/exit_transforms.cpp

```diff
@@ -310,41 +416,24 @@ struct ExitTransformer {
 
   // we create one uninitialized value per type, cache it here and reuse it
   std::unordered_map<TypePtr, Value*> unit_values_;
+
+  // can either be LoopContinuation/ReturnStmt
+  Symbol current_exit_kind_;
   Value* true_val_;
   Value* false_val_;
 
+  // the block that we are currently targeting in the transform.
```

Comment:
Might be helpful to provide some context here - it's not obvious what "targeting" means unless you already know what the transformation is doing.